### PR TITLE
FISH-6406 : keystore & cacerts migrated to PKCS12 format

### PR DESCRIPTION
**Pull request by luiseufrasio**
## Description
cacerts and truststore migrated to PKCS12 format

### Blockers
none

### New tests
none

### Testing Performed
https://jenkins.payara.fish/view/Jakarta%20TCK%20Testing/job/TCK-Suite/706
https://jenkins.payara.fish/view/Jakarta%20TCK%20Testing/job/TCK-Suite/707
https://jenkins.payara.fish/view/Jakarta%20TCK%20Testing/job/TCK-Suite/708
https://jenkins.payara.fish/view/Jakarta%20TCK%20Testing/job/TCK-Suite/698

### Testing Environment
Jenkins

## Documentation
none

## Notes for Reviewers
The last test fails with a database issue regarding to the pool.
The same problem occurs on Payara6 branch with TCK master branch

### CODE REVIEW

**Comment by breakponchito** [comment]
Jenkins test please

File: appserver/appclient/server/core/src/main/java/org/glassfish/appclient/server/core/jws/servedcontent/ASJarSigner.java [hunk 1]
```diff
@@ -87,7 +87,7 @@ public class ASJarSigner implements PostConstruct {
     public static final String USER_SPECIFIED_ALIAS_PROPERTYNAME = "com.sun.aas.jws.signing.alias";
 
     /** keystore type for JKS keystores */
-    private static final String JKS_KEYSTORE_TYPE_VALUE = "jks";
+    private static final String    JKS_KEYSTORE_TYPE_VALUE = "jks";
```
Comment (breakponchito) [inline appserver/appclient/server/core/src/main/java/org/glassfish/appclient/server/core/jws/servedcontent/ASJarSigner.java:90]: Should this need to be removed?

**Comment by breakponchito** [comment]
Jenkins test please

**Comment by breakponchito** [comment]
Jenkins test please

File: appserver/appclient/server/core/src/main/java/org/glassfish/appclient/server/core/jws/servedcontent/ASJarSigner.java (hunk 1 above)
Comment (luiseufrasio) [reply appserver/appclient/server/core/src/main/java/org/glassfish/appclient/server/core/jws/servedcontent/ASJarSigner.java]: done!

**Comment by luiseufrasio** [comment]
> I get the same NPE as Jenkins locally
> 
> ```
> [INFO] Running fish.payara.microprofile.jaxrs.client.ssl.RestClientSslContextAliasListenerTest
> [ERROR] Tests run: 2, Failures: 0, Errors: 2, Skipped: 0, Time elapsed: 0.365 s <<< FAILURE! - in fish.payara.microprofile.jaxrs.client.ssl.RestClientSslContextAliasListenerTest
> [ERROR] fish.payara.microprofile.jaxrs.client.ssl.RestClientSslContextAliasListenerTest.restClientAliasPropertySslContextTest  Time elapsed: 0.317 s  <<< ERROR!
> java.lang.NullPointerException
>         at fish.payara.microprofile.jaxrs.client.ssl.RestClientSslContextAliasListenerTest.getKeyStore(RestClientSslContextAliasListenerTest.java:125)
>         at fish.payara.microprofile.jaxrs.client.ssl.RestClientSslContextAliasListenerTest.getManagers(RestClientSslContextAliasListenerTest.java:131)
> ```

fixed

> I get the same NPE as Jenkins locally
> 
> ```
> [INFO] Running fish.payara.microprofile.jaxrs.client.ssl.RestClientSslContextAliasListenerTest
> [ERROR] Tests run: 2, Failures: 0, Errors: 2, Skipped: 0, Time elapsed: 0.365 s <<< FAILURE! - in fish.payara.microprofile.jaxrs.client.ssl.RestClientSslContextAliasListenerTest
> [ERROR] fish.payara.microprofile.jaxrs.client.ssl.RestClientSslContextAliasListenerTest.restClientAliasPropertySslContextTest  Time elapsed: 0.317 s  <<< ERROR!
> java.lang.NullPointerException
>         at fish.payara.microprofile.jaxrs.client.ssl.RestClientSslContextAliasListenerTest.getKeyStore(RestClientSslContextAliasListenerTest.java:125)
>         at fish.payara.microprofile.jaxrs.client.ssl.RestClientSslContextAliasListenerTest.getManagers(RestClientSslContextAliasListenerTest.java:131)
> ```

NPE fixed

**Comment by luiseufrasio** [comment]
> I get the same NPE as Jenkins locally
> 
> ```
> [INFO] Running fish.payara.microprofile.jaxrs.client.ssl.RestClientSslContextAliasListenerTest
> [ERROR] Tests run: 2, Failures: 0, Errors: 2, Skipped: 0, Time elapsed: 0.365 s <<< FAILURE! - in fish.payara.microprofile.jaxrs.client.ssl.RestClientSslContextAliasListenerTest
> [ERROR] fish.payara.microprofile.jaxrs.client.ssl.RestClientSslContextAliasListenerTest.restClientAliasPropertySslContextTest  Time elapsed: 0.317 s  <<< ERROR!
> java.lang.NullPointerException
>         at fish.payara.microprofile.jaxrs.client.ssl.RestClientSslContextAliasListenerTest.getKeyStore(RestClientSslContextAliasListenerTest.java:125)
>         at fish.payara.microprofile.jaxrs.client.ssl.RestClientSslContextAliasListenerTest.getManagers(RestClientSslContextAliasListenerTest.java:131)
> ```

NPE fixed

**Review by Pandrex247 — APPROVED** [review]
LGTM!

We will need to coordinate this with all of the other PRs

**Comment by luiseufrasio** [comment]
jenkins test please

**Pull request merged**